packages/website/src/templates/doc-page.js

```diff
@@ -24,7 +24,20 @@ export const components = {
   a: RC.Link,
   h1: props => <RC.Heading {...props} fontWeight="bold" size={3} mt={8} mb={4} />,
   h2: props => <RC.Heading {...props} as="h2" fontWeight="bold" size={2} mt={8} mb={4} />,
-  h3: props => <RC.Heading {...props} as="h3" fontWeight="bold" size={1} mt={8} mb={4} />,
+  h3: props => (
+    <RC.Heading {...props} as="h3" fontWeight="bold" size={1} mt={7} mb={1} lineHeight={1} />
+  ),
+  p: props => (
+    <RC.Text
+      {...props}
+      as="p"
+      size={3}
+      mt={0}
+      mb={0}
+      lineHeight={1}
+      style={{ color: 'hsl(208, 14%, 46%)' }}
```

Comment:
What's this inline color? The Radix `Text` Component accepts a `textColor` prop which is bound to the theme.

Reply:
I tried using `color={grays.5}` but it didn't work obviously. I'll fix it.